pkg/cli/interactive_tests/test_notice.tcl

```diff
@@ -7,19 +7,36 @@ source [file join [file dirname $argv0] common.tcl]
 spawn $argv demo --empty
 eexpect root@
 
-start_test "Test creating a view without TEMP set on a TEMP TABLE notifies it will become a TEMP VIEW."
-send "SET experimental_enable_temp_tables = true; CREATE TEMP TABLE temp_view_test_tbl(a int); CREATE VIEW temp_view_test AS SELECT a FROM temp_view_test_tbl;\r"
-eexpect "NOTICE: view \"temp_view_test\" will be a temporary view"
+start_test "Test that notices always appear at the end after all results."
+send "SELECT IF(@1=4,crdb_internal.notice('hello'),@1) AS MYRES FROM generate_series(1,10);\r"
+eexpect myres
+eexpect 1
+eexpect 10
+eexpect "10 rows"
+eexpect "NOTICE: hello"
+eexpect root@
+
+# Ditto with multiple result sets. Notices after all result sets.
+send "SELECT crdb_internal.notice('hello') AS STAGE1;"
+send "SELECT crdb_internal.notice('world') AS STAGE2;\r"
+eexpect stage1
+eexpect stage2
+eexpect "NOTICE: hello"
+eexpect "NOTICE: world"
+eexpect root@
 end_test
 
 start_test "Check dropping an index prints a message about GC TTL."
 send "CREATE TABLE drop_index_test(a int); CREATE INDEX drop_index_test_index ON drop_index_test(a); DROP INDEX drop_index_test_index;\r"
-eexpect "NOTICE: index \"drop_index_test_index\" will be dropped asynchronously and will be complete after the GC TTL"
+eexpect "NOTICE: the data for dropped indices is reclaimed asynchronously"
```

Comment:
i think this should be "indexes", not "indices"... https://www.postgresql.org/docs/9.1/indexes.html....

Reply:
thanks. will fix